nx.json

```diff
@@ -15,24 +15,24 @@
             "dependsOn": ["^build:lib"],
             "inputs": [
                 "^prod",
-                "{workspaceRoot}/tsconfig.json",
+                "{workspaceRoot}/tsconfig.base.json",
                 "{workspaceRoot}/tsconfig.lib.json"
             ],
             "outputs": ["{projectRoot}/lib", "{projectRoot}/build"],
             "cache": true
         },
         "type-check": {
-            "dependsOn": ["^build:lib", "^type-check"],
+            "dependsOn": ["^build:lib", "^type-check", "build:lib"],
```

Comment:
Is this necessary? Seems redundant

Reply:
It's two different things `^build:lib` mean run that build:lib command packages that this packages needs. Without `^` I am defining that it should run build:lib before type-check for current package (this did not work before).